client/app/components/packages/filed-eas/edit.hbs

```diff
@@ -21,5 +21,19 @@
         {{optionset 'package' 'statuscode' 'label' @package.statuscode}}
       </p>
     </section>
+
+    {{#if @package.dcpPackagenotes}}
+      <section class="form-section">
+        <p>
+          [Instructions around how the EAS package includes Draft EAS, Filed EAS (which is dependent on the CEQR Invoice Payment Questionnaire), and Draft Scope of Work.
```

Comment:
Need to remove this "[Instructions around how the"

Reply:
anything I can do to help get the language for this?